src/views/ont/ONTPoolIndex.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <DataFetcher :fetcher="fetchOntPools">
     <FilterCard :fetcher="fetchOntPools" :filter-options="filterOptions" />
+    <div class="flex mt-2 mb-3 border" />
```

Comment:
Instead of adding this to all pages could we move it into the FilterCard component as a wrapper?

Reply:
Yes, makes sense in the way  how it is used currently and it is required in all  calling contexts. I didn't do that because having a split-bar below doesn't actually can be considered as this component styling but rather belongs to the choice at parent level. 
 I will move it to FilterCard and probably make it as an customisable style based on a prop - does that sound good?

Reply:
Yeah I guess it does sit in a bit of a grey area. Yes I think adding custom class props or something would be a good solution.

Reply:
Done